trace/exporterutil/doc.go

```diff
@@ -0,0 +1,16 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package exporterutil contains utilities to support trace exporting.
+package exporterutil
```

Comment:
I'd discourage us to name this package exportutil.

It is very likely trace package will have more utilities. That's why trace/traceutil is a better place.

Reply:
Can you give an example? I would prefer to keep this exporter-focussed because I think the most likely candidates for inclusion in future are also exporter-related: specifically, I would like to pull all the batching logic out of the individual exporters and put it here.

Reply:
Then you should start an exporter package somewhere. This package doesn't fit under the trace namespace.

Reply:
okay moving to the top level

Reply:
I ended up renaming to traceutil as you originally suggested